index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+
+</head>
+
+
+<body>
+    <main id = "main">
+        <h1 id = "title">Youssef and Gustavo</h1>
+        <p> Master Yoda: "Do... or do not, There is no try"</p>
+        <div id = "img-div">
+            <img id="image" src="picture.jpg" alt="">
+        </div>
+        <div id = "img-caption">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Possimus iusto officiis ullam tenetur est libero dolorem vel 
+                rerum vitae porro, architecto dolore, iure voluptates quo. Explicabo ipsa ut mollitia eaque.</p>
+
+
+        </div>
+        <div>
+            <h3 id="tribute-info">The Life of Yoda, who lives more than 800 years</h3>
+
+            <ul id = "text">
+                <li id = "text_1">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate dolore aspernatur aliquam reiciendis ipsam distinctio natus animi vero incidunt 
+                    maxime architecto, cum dignissimos id maiores saepe ratione, ex nihil magnam.</li>
+                <li id = "text_2">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor maiores dicta, fugit soluta delectus id quos culpa? Nulla hic dicta assumenda 
+                    eos cumque molestias, doloremque sint ipsum nobis veniam quam.</li>  
+                <li id = "text_3">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quos reprehenderit iure omnis, rem asperiores repellendus maiores impedit minima 
+                    voluptas culpa sit enim perspiciatis nemo veniam est ipsam totam modi fugiat.</li>
+            </ul>
+            <a id="tribute-link" target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Yoda">MASTER YODA</a>
+
+        </div>
+        
+    </main>
+    <script src = "dom.js"></script>
+    
```

Comment:
I would like to advise you to consider no left **blank** lines in the code, so that can be more ordered and clean for every person who sees your master coding skills 💯 correctly.

Reply:
@danmainah sorry if I don`t make myself understood. Thinking in Spanish and writing in English is a struggle for me right now